nornir_nautobot/plugins/tasks/dispatcher/checkpoint_gaia.py

```diff
@@ -0,0 +1,51 @@
+"""nornir dispatcher for checkpoint_gaia."""
+
+from nornir.core.task import Result, Task
+
+from nornir_nautobot.plugins.tasks.dispatcher.default import NapalmDefault, NetmikoDefault
+from nornir_nautobot.exceptions import NornirNautobotException
+
+NETMIKO_DEVICE_TYPE = "linux"
```

Comment:
In theory this will come directly from the nornir inventory in the platform definition. Would like to avoid hardcoding it here.